### PR TITLE
Marketplace storefront

### PR DESCRIPTION
**Pull request by andriitserkovnyi**
## PR Description

https://spryker.atlassian.net/browse/AC-1263

Created a Marketplace Storefront document.

## Checklist
- [x ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/marketplace/user/intro-to-the-spryker-marketplace/marketplace-storefront.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+template: concept-topic-template
+---
+
+*Marketplace Storefront* is a web application designed for the [marketplace business model](/docs/marketplace/user/intro-to-the-spryker-marketplace/{{ site.version }}/marketplace-concept.html). Based on the regular Storefront, it supports all Spryker Commerce OS features. Marketplace functionality enables buyers to browse for products, view and purchase them across different merchants.
+
+In every section of the Marketplace Storefront, products are either sorted or customers can filter them by merchants.
+
+## Catalog and Search pages
+
+On the *Catalog* and *Search* pages, the left-side navigation menu contains the *Merchant* filter. Customers can select one or more merchants from which to view products.
```
Comment (mikost7) [inline docs/marketplace/user/intro-to-the-spryker-marketplace/marketplace-storefront.md:13]:
```suggestion
On the *Catalog* and *Search* pages, the left-side navigation menu contains the *Merchant* filter. Customers can select one or more merchants from which to view products or offers.
```

File: docs/marketplace/user/intro-to-the-spryker-marketplace/marketplace-storefront.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+In the *Sold by* section, when a customer opens a *Product Details* page, the product is pre-selected, and they can switch to any other offer. In the list, the pre-selected [marketplace product](/docs/marketplace/user/features/{{ page.version }}/marketplace-product-feature-overview.html) always appears first, followed by offers in ascending order of price. If the product is not available, but there are offers for it, the cheapest offer is pre-selected.
+
+The sorting of the *Sold by* section is configurable on a project level.
+
+The product price on top of the *Product Details* page is taken from the selected merchant's product or offer. When a customer selects a product or offer in the *Sold by* section, the page refreshes showing selected merchant's price.
+
+## Cart page
+
+On the *Cart* page, a merchant relation is displayed for each product. The merchant names are clickable and lead to the merchant profile pages.
```
Comment (mikost7) [inline docs/marketplace/user/intro-to-the-spryker-marketplace/marketplace-storefront.md:31]:
```suggestion
On the *Cart* page, a merchant reference is displayed for each product. The merchant names are clickable and lead to the merchant profile pages.
```

File: docs/marketplace/user/intro-to-the-spryker-marketplace/marketplace-storefront.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+![Shipment step](https://spryker.s3.eu-central-1.amazonaws.com/docs/Marketplace/user+guides/Intro+to+the+Spryker+Marketplace/Marketplace+Storefront/shipment-step.png)
+
+At the Summary checkout step,  merchant relation is displayed for each product. The merchant names are clickable and lead to the merchant profile pages. If a customer is ordering products from multiple merchants, the products are automatically grouped by merchants, which also represents the order's shipments.
+
+![Summary step](https://spryker.s3.eu-central-1.amazonaws.com/docs/Marketplace/user+guides/Intro+to+the+Spryker+Marketplace/Marketplace+Storefront/summary-step.png)
+
+
+### Order Details pages
+
+On the *Order Details* page, a merchant relation is displayed for each product. The merchant names are clickable and lead to the merchant profile pages. If a customer ordered products from multiple merchants, the products are grouped by merchants, which also represents the order's shipments.
```
Comment (mikost7) [inline docs/marketplace/user/intro-to-the-spryker-marketplace/marketplace-storefront.md:53]:
```suggestion
On the *Order Details* page, a merchant reference is displayed for each product. The merchant names are clickable and lead to the merchant profile pages. If a customer ordered products from multiple merchants, the products are grouped by merchants, which also represents the order's shipments.
```

File: docs/marketplace/user/intro-to-the-spryker-marketplace/marketplace-storefront.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+![Order Details page](https://spryker.s3.eu-central-1.amazonaws.com/docs/Marketplace/user+guides/Intro+to+the+Spryker+Marketplace/Marketplace+Storefront/order-details-page.png)
+
+If the customer clicks **Reorder all**, the items are added to their current cart with the merchant relations from the order.
```
Comment (mikost7) [inline docs/marketplace/user/intro-to-the-spryker-marketplace/marketplace-storefront.md:57]:
```suggestion
If the customer clicks **Reorder all**, the items are added to their current cart with the merchant references from the order.
```

File: docs/marketplace/user/intro-to-the-spryker-marketplace/marketplace-storefront.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+### Marketplace Wishlists
+
+When a customer adds a product to a wishlist, the product is added with the merchant relation selected in the *Sold by* section.
+
+On the page of the wishlist, a merchant relation is displayed for each product. Merchant names are clickable and lead to the merchant profile pages.
+
+![Wishlist page](https://spryker.s3.eu-central-1.amazonaws.com/docs/Marketplace/user+guides/Intro+to+the+Spryker+Marketplace/Marketplace+Storefront/wishlist-page.png)
+
+If, after a product was added to a wishlist, a merchant becomes inactive, the merchant relation is no longer displayed for the product and the customer can't add it to cart. If the merchant becomes active again, the relation is displayed and the customer can add the product to cart.
```
Comment (mikost7) [inline docs/marketplace/user/intro-to-the-spryker-marketplace/marketplace-storefront.md:76]:
```suggestion
If, after a product was added to a wishlist, a merchant becomes inactive, the merchant reference is no longer displayed for the product and the customer can't add it to cart. If the merchant becomes active again, the relation is displayed and the customer can add the product to cart.
```

File: docs/marketplace/user/intro-to-the-spryker-marketplace/marketplace-storefront.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+
+If a product in a wishlist is out of stock, an [alternative product](https://documentation.spryker.com/docs/alternative-products-overview) is displayed. A merchant relation is displayed for the alternative product. Currently, merchant offers are not supported by alternative products, so the product displayed is a merchant or a regular product.
```
Comment (mikost7) [inline docs/marketplace/user/intro-to-the-spryker-marketplace/marketplace-storefront.md:78]:
```suggestion
If a product in a wishlist is out of stock, an [alternative product](https://documentation.spryker.com/docs/alternative-products-overview) can be displayed. A merchant reference is displayed for the alternative product. Currently, merchant offers are not supported by alternative products, so only merchant products can be displayed there.
```

File: docs/marketplace/user/intro-to-the-spryker-marketplace/marketplace-storefront.md [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+title: Marketplace Storefront
+description: This document describes the peculiarities of the Marketplace Storefront application.
+template: concept-topic-template
+---
+
+*Marketplace Storefront* is a web application designed for the [marketplace business model](/docs/marketplace/user/intro-to-the-spryker-marketplace/{{ site.version }}/marketplace-concept.html). Based on the regular Storefront, it supports all Spryker Commerce OS features. Marketplace functionality enables buyers to browse for products, view and purchase them across different merchants.
+
+In every section of the Marketplace Storefront, products are either sorted or customers can filter them by merchants.
+
+## Catalog and Search pages
```
Comment (mikost7) [inline docs/marketplace/user/intro-to-the-spryker-marketplace/marketplace-storefront.md:11]: can we also mention here the fact that if searching by merchant name in the search field, the products and offers of the relevant merchant will be displayed?

**Review by mikost7 — CHANGES_REQUESTED** [review]
pls check comments

File: docs/marketplace/user/intro-to-the-spryker-marketplace/marketplace-storefront.md (hunk 7 above)
Comment (gibmirmilch) [reply docs/marketplace/user/intro-to-the-spryker-marketplace/marketplace-storefront.md:11]: added

**Pull request merged**